syft/src/main/java/org/openmined/syft/data/KTensor.kt

```diff
@@ -1,11 +1,19 @@
-package org.openmined.syft.demo.federated.domain
+package org.openmined.syft.data
 
-data class Batch(val flattenedArray: FloatArray, val shape: LongArray) {
+import org.pytorch.IValue
+import org.pytorch.Tensor
+
+/**
+ * Wrapper of data and shape
+ * @property flattenedArray the actual data
+ * @property shape data shape
+ * */
+data class KTensor(val flattenedArray: FloatArray, val shape: LongArray) {
```

Comment:
SyftTensor is perfect if we would not need the name for something else.
But SyftTensor is the right choice.

Reply:
What functionality are we adding over a usual tensor for needing another class? as far as I can see, we have shape and data array. So SyftTensor at this moment is a serialization tensor class so it can be renamed to SyftProtoTensor.